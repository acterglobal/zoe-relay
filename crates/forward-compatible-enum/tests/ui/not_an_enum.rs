use forward_compatible_enum::ForwardCompatibleEnum;

#[derive(ForwardCompatibleEnum)]  // Should fail - not an enum!
pub struct NotAnEnum {
    field: u32,
}