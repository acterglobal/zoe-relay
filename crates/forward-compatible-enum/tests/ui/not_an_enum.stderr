error: ForwardCompatibleEnum can only be applied to enums
 --> tests/ui/not_an_enum.rs:4:1
  |
4 | / pub struct NotAnEnum {
5 | |     field: u32,
6 | | }
  | |_^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/not_an_enum.rs:6:2
  |
6 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/not_an_enum.rs`
